spec/spec_helper.rb

```diff
@@ -0,0 +1,5 @@
+require 'chefspec'
```

Comment:
Would these dependencies need to be added to the `Gemfile` so that they can be installed to run the specs?